### PR TITLE
Update organization.md

### PR DESCRIPTION
**Pull request by jmoody02**
Additions needed for new org registration

### CODE REVIEW

File: organization.md [hunk 1]
```diff
@@ -26,13 +28,27 @@ This section shows how developers can make use of the features of this specifica
...
+    "biling_address": {
+        "street": ["2111 Snow Road"],
+        "city": "Parma",
+        "region": "US-OH",
+        "postal_code": "44134",
+        "country": "US"
+    },
+    "phone_number": "2222222222",
+    "fax_number": "2222222222",
+    "federal_tax_id": "123456789",
```
Comment (cwardzala) [inline organization.md:46]: Is there any legal issues with storing this value in plain text?

File: organization.md [hunk 2]
```diff
@@ -26,13 +28,27 @@ This section shows how developers can make use of the features of this specifica
...
+    "tax_exempt": true,
+    "tax_exempt_number": "123456789",
```
Comment (cwardzala) [inline organization.md:48]: Is there any legal issues with storing this value in plain text?

File: organization.md [hunk 3]
```diff
@@ -26,13 +28,27 @@ This section shows how developers can make use of the features of this specifica
...
+    "tax_exempt_form_uri": "",
```
Comment (cwardzala) [inline organization.md]: Not sure we need this data

File: organization.md [hunk 4]
```diff
@@ -26,13 +28,27 @@ This section shows how developers can make use of the features of this specifica
...
+    "notes": "",
```
Comment (cwardzala) [inline organization.md:49]: How are the current "notes" used in NetSuite? I dont see us needing this data for anything on the web.

File: organization.md [hunk 5]
```diff
@@ -10,10 +10,12 @@ This section shows how developers can make use of the features of this specifica
 ```json
 {
     "id": "01fc7fce-bf78-4b50-903a-fb3b760f5b30",
+    "netsuite_id": "999999",
```
Comment (cwardzala) [inline organization.md]: lets call this "internal_id" that way if we move off NetSuite its not tied to that system.

File: organization.md (hunk 1 above)
Comment (cwardzala) [inline organization.md:46]: Also I dont know that we need this info on the web, is this stored in NetSuite?

File: organization.md (hunk 2 above)
Comment (cwardzala) [inline organization.md:48]: I dont know that we need this info on the web, is this stored in NetSuite?

File: organization.md (hunk 4 above)
Comment (jmoody02) [inline organization.md:49]: It's customer requirements in Netsuite, and I just pulled it from the New Organizational form mockup.

File: organization.md (hunk 3 above)
Comment (jmoody02) [inline organization.md]: I can't tell which line each comment is referring to, but I pulled them all from the New Organization Form mock up.

**Pull request merged**